.rubocop_todo.yml

```diff
@@ -321,6 +317,12 @@ Style/NumericPredicate:
     - 'lib/bundler/source/git/git_proxy.rb'
     - 'lib/bundler/source/path.rb'
 
+# Offense count: 192
+# Cop supports --auto-correct.
+# Configuration parameters: PreferredDelimiters.
+Style/PercentLiteralDelimiters:
```

Comment:
Thanks for your review.

Sure. It seems good to enable Style/PercentLiteralDelimiters cop according to the default setting of RuboCop.
Https://github.com/bbatsov/rubocop/blob/v0.48.1/config/enabled.yml#L651-L654

I applied Style/PercentLiteralDelimiters cop to the project by running `rubocop -a --only Style/PercentLiteralDelimiters` and `rubocop --auto-gen-config`. That commit is 4f7ffab5fc42ad54b3e040aedaf8c933806104d7.

How about that?